src/saving/SaveHelper.cs

```diff
@@ -295,20 +296,20 @@ public static void DeleteExcessSaves(string nameStartsWith, int maximumCount)
     }
 
     /// <summary>
-    ///   Deletes all saves with the given prefix except the latest one and returns the list of saves deleted
+    ///   Deletes all saves with the given regex expression except the
+    ///   latest one if deleteLatest is false and returns the list of saves deleted
```

Comment:
This line really begs to be separate parameter and `<returns></returns>` xml elements.